app/code/core/Mage/Catalog/Model/Indexer/Url.php

```diff
@@ -237,9 +237,12 @@ protected function _processEvent(Mage_Index_Model_Event $event)
                  $urlModel->refreshProductRewrite($productId);
             }
         }
-        if (isset($data['rewrite_category_ids'])) {
+
+        $rewriteCategoryIds = $data['rewrite_category_ids'] ?? null;
```

Comment:
I love the null coalescing operator as much as the next PHP guy, but unfortunately we haven't abandoned PHP 5.x yet..

Reply:
this is for our internal project.
Do you want for me to update to be 5.x+ compatible ?

Reply:
Yes, that would be great! Thanks

Reply:
Raised https://github.com/OpenMage/magento-lts/pull/582 for that